### PR TITLE
Is/fix/flutter android push notification

### PR DESCRIPTION
**Pull request by isaacakakpo1**
[ENGDESK-28515 - Ticket Description.](https://telnyx.atlassian.net/browse/ENGDESK-28515)

Implemented HandlepushNotification to handle push notifications

## ✋ Manual testing
1. Push Calls Work

### CODE REVIEW

**Review by imranbilal1 — APPROVED** [review]

File: ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -492,14 +496,15 @@
...
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = Runner/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.telnyx.telnyxFlutterWebrtc;
+				PRODUCT_BUNDLE_IDENTIFIER = com.isaac.telnyxFlutterWebrtc;
```
Comment (RJSkorski) [inline ios/Runner.xcodeproj/project.pbxproj:507]: is that correct?

File: ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -516,14 +521,15 @@
...
```
Comment (RJSkorski) [inline ios/Runner.xcodeproj/project.pbxproj:532]: is that correct?

Comment (isaacakakpo1) [reply ios/Runner.xcodeproj/project.pbxproj:532]: Yeah, I still don't have access to com.telnyx.telnyxFlutterWebrtc gor Firebase. I had to use my own. I'll just make the correction . I'll have to change it to mine everytime I want to test the app

Comment (isaacakakpo1) [reply ios/Runner.xcodeproj/project.pbxproj:532]: It's generated code as well

**Pull request merged**